cypress/suites/login.suite.ts

```diff
@@ -239,6 +235,30 @@ export default {
         expectToBeOnDashboard();
       });
 
+      /* eslint-disable-next-line mocha-cleanup/asserts-limit */
+      it.only("should allow a user to go back to the beginning from any step", () => {
+        // go to second step
+        expectClickGenerateWalletButton();
+        cy.get("#onboarding_back_button").trigger("click");
+
+        // go to third step
+        expectClickGenerateWalletButton();
+        cy.get("textarea").invoke("val").then(stringToWordArray).as("seedWords");
+        cy.get("label").contains("I have backed up my seed phrase").click();
+        cy.get("#onboarding_back_button").trigger("click");
+
+        // go to final step
+        cy.get("textarea").invoke("val").then(stringToWordArray).as("seedWords");
```

Comment:
would be pretty slick to make a function which could take the user to a step based on a passed in argument so these could look like:

`goToOnboardingStep(2)` for the second onboarding step